src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { useCallback, useMemo } from "react";
+import noop from "lodash/noop";
```

Comment:
We don't need to import from a library for this.. the whole implementation is `() => undefined`

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1225/commits/6a8bb74a884fc0df9e55f819a976c8214b771741